kivy/tests/test_graphics.py

```diff
@@ -109,3 +109,19 @@ def test_identity_creation(self):
         from kivy.graphics import LoadIdentity
         mat = LoadIdentity()
         self.assertTrue(mat.stack)
+
+
+class CallbackInstructionTest(GraphicUnitTest):
+
+    def test_from_kv(self):
+        from kivy.lang import Builder
+
+        root = Builder.load_string("""
+Widget:
+    canvas:
+        Callback:
+            callback: lambda __: setattr(self, 'callback_test', 'TEST')
+""")
```

Comment:
nit picking: you could use textwrap.dedent here to keep the indentation consistent here.

Reply:
thanks :) I've fixed that.